packages/react/src/FAQ/FAQGroup.tsx

```diff
@@ -73,8 +74,13 @@ function _FAQGroup({children, id, defaultSelectedIndex = 0, ...rest}: FAQGroupPr
         child => React.isValidElement(child) && child.type === FAQ.Heading,
       )
 
+      const tabContents = React.isValidElement(GroupHeadingChild) && GroupHeadingChild.props.children
+
+      const providedTabAttributes = tabAttributes?.(tabContents, index)
+
       return (
         <Button
+          {...providedTabAttributes}
```

Comment:
Was it intended to put this at the top vs bottom? I.e. to avoid overriding the default values? On one hand, I love the intrinsic guardrails, but also wondering if this is going to be confusing for anyone that is trying to unsuccessfully set override props

Reply:
Yes it was intentional and I was just erring on the side of guardrails. 

I'm happy moving it to the bottom (or even to somewhere in the middle) if you think there's value in allowing some/all props to be overriden

Reply:
all good, let's leave it like this. thanks for clarifying.. 👍